### PR TITLE
BE: Make gh version check timeout configurable

### PR DESCRIPTION
**Pull request by wernerdv**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)

Fixed https://github.com/kafbat/kafka-ui/issues/477.

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [ ] No need to
- [x] Manually (please, describe, if necessary)
- [x] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/kafbat/kafka-ui/blob/main/.github/CONTRIBUTING.md) and [Code of Conduct](https://github.com/kafbat/kafka-ui/blob/main/.github/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by wernerdv** [comment]
@Haarolean Hello, PTAL.

**Comment by wernerdv** [comment]
@Haarolean PTAL.
Is there anything I can do to improve this PR?

File: api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java [hunk 1]
```diff
@@ -32,6 +34,7 @@
 public abstract class AbstractIntegrationTest {
   public static final String LOCAL = "local";
   public static final String SECOND_LOCAL = "secondLocal";
+  protected static final int timeout = 100;
```
Comment (Haarolean) [inline api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java:37]: I belive we could get rid of this variable as it doesn't seem reusable anyway

**Review by Haarolean — CHANGES_REQUESTED** [review]

File: api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java (hunk 1 above)
Comment (wernerdv) [reply api/src/test/java/io/kafbat/ui/AbstractIntegrationTest.java]: Done

File: api/src/test/java/io/kafbat/ui/service/ApplicationInfoServiceTest.java [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+
+public class ApplicationInfoServiceTest extends AbstractIntegrationTest {
+  @Autowired
+  private ApplicationInfoService service;
+
+  @Test
+  public void testCustomGithubReleaseInfoTimeout() {
+    assertEquals(100, service.githubReleaseInfo().getGithubApiMaxWaitTime());
```
Comment (Haarolean) [inline api/src/test/java/io/kafbat/ui/service/ApplicationInfoServiceTest.java:15]: A test might be an overkill for a feature like this, but thanks! 😄

**Review by Haarolean — APPROVED** [review]

**Comment by Haarolean** [comment]
@wernerdv thanks for your contribution!

**Pull request merged**